.github/RELEASE_TEMPLATES/cli_release_checklist.md

```diff
@@ -0,0 +1,18 @@
+---
+title: CLI Release {{ env.RELEASE_NAME }}
+labels: release
+---
+## Requirements
+- [ ] ⚠️ Ensure the release process has succeeded before proceeding
+
+## Github
+- [ ] Update [CHANGELOG.md]({{ env.CHANGELOG_URL }})
+- [ ] Update [README.md]({{ env.README_URL }})
+- [ ] Publish [Release]({{ env.RELEASE_DRAFT }})
+- [ ] Merge Release PR ({{ env.RELEASE_PR }})
```

Comment:
oh sorry, we also need to add a RELEASE_PR environment variable.

Reply:
Good catch! I mistakenly thought the CLI release process doesn't create a version update PR. (Worse, I removed the checklist item in Core, not CLI.)